### ISSUE TITLE
HydroBot - Social media Detection

### ISSUE THREAD

**Issue by kberey** [HydroBlockchain/Community-Brainstorming#57]
What is your Suggestion for the Hydro Community?

If the people visit the social media of hydro will have points/hydro. The logo of bot is Hydro they need to download on IOS or google play. The hydro bot detects if you like/share/put a comment on facebook. Hydro bot detects if you tweeted or like the page of twitter, Hydro bot will detect if you put clap on Medium. Hydro bot must be detected only 1 accounts with KYC. 

The more you earn, subscribe, the more you earn on hydro bot.

![hydro robot](https://user-images.githubusercontent.com/39163694/59373960-c4ae4e80-8d18-11e9-8ee7-c514fa5a02bb.png)

**Comment by ansteadm** [comment]
This would require for Hydro to basically be deposited by someone to earn it- sort of like a faucet. Is this was you're proposing?

**Comment by kberey** [comment]
Yes. Project Hydro fund the Hydro bot first like 1Million hydro token  Hydro Bot --> People --> Earnings via social media --> exchange market--> hydro bot --> repeat

(All earnings in the Hydro via traffic,social media,trends will fund again the hydro bot) USD - Hydro. again in the market. This is what I think. @ansteadm

**Comment by CAEGlobal** [comment]
It's interesting, I love the idea, I think the goal here is to encourage active and engaged social media community.

**Comment by kberey** [comment]
> It's interesting, I love the idea, I think the goal here is to encourage active and engaged social media community.

Exactly!  thanks for your comment.

**Comment by Tweevie** [comment]
Nice idea for community interaction and low key marketing.

Sounds quite similar to those airdrops where you have to complete tasks like follow, share, like, comment on number of platforms and earn tokens.

**Comment by kberey** [comment]
> Nice idea for community interaction and low key marketing.
> 
> Sounds quite similar to those airdrops where you have to complete tasks like follow, share, like, comment on number of platforms and earn tokens.

Yes the airdrops is for the beginning of the project right? or if there's an event/giveaways. this hydro Bot will detect the rest of the project. even there's no giveaways or events. Also I would like to add the Tip/donate if we can do that.

**Comment by DANILORCORTAGA** [comment]
Nice very nice concept to gain more in community

**Comment by ebunayo** [comment]
This is a wonderful idea that can be applied to every traditional marketing system. Can you draw up technical requirements?
